From 054a7dd159f32b833133f23de453346359c3fd60 Mon Sep 17 00:00:00 2001
From: Qiuxu Zhuo <qiuxu.zhuo@intel.com>
Date: Wed, 15 Dec 2021 03:48:11 -0500
Subject: [PATCH 32/41] EDAC/igen6: Add another Intel Alder Lake SoC support

Add another Intel Alder Lake SoC compute die ID 0x460a for EDAC
support.

Signed-off-by: Qiuxu Zhuo <qiuxu.zhuo@intel.com>
Tested-by: Wen Jin <wen.jin@intel.com>
---
 drivers/edac/igen6_edac.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/edac/igen6_edac.c b/drivers/edac/igen6_edac.c
index 6eb0e0bb8616..1e3c7eef1ccb 100644
--- a/drivers/edac/igen6_edac.c
+++ b/drivers/edac/igen6_edac.c
@@ -221,6 +221,7 @@ static struct work_struct ecclog_work;
 #define DID_ADL_SKU2	0x4602
 #define DID_ADL_SKU3	0x4621
 #define DID_ADL_SKU4	0x4641
+#define DID_ADL_SKU5	0x460a
 
 /* Compute die IDs for Alder Lake-N with IBECC */
 #define DID_ADL_N_SKU1	0x4614
@@ -448,6 +449,7 @@ static const struct pci_device_id igen6_pci_tbl[] = {
 	{ PCI_VDEVICE(INTEL, DID_ADL_SKU2), (kernel_ulong_t)&adl_cfg },
 	{ PCI_VDEVICE(INTEL, DID_ADL_SKU3), (kernel_ulong_t)&adl_cfg },
 	{ PCI_VDEVICE(INTEL, DID_ADL_SKU4), (kernel_ulong_t)&adl_cfg },
+	{ PCI_VDEVICE(INTEL, DID_ADL_SKU5), (kernel_ulong_t)&adl_cfg },
 	{ PCI_VDEVICE(INTEL, DID_ADL_N_SKU1), (kernel_ulong_t)&adl_n_cfg },
 	{ PCI_VDEVICE(INTEL, DID_ADL_N_SKU2), (kernel_ulong_t)&adl_n_cfg },
 	{ PCI_VDEVICE(INTEL, DID_ADL_N_SKU3), (kernel_ulong_t)&adl_n_cfg },
-- 
2.17.1

